tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -173,7 +173,9 @@ Spawn Notebook With Arguments  # robocop: disable
          Exit For Loop If  ${spawn_fail} == False
          Click Element  xpath://span[@id='jupyterhub-logo']
       ELSE
+         Run Keyword If  ${index} == ${retries}  Fail
          Click Element  xpath://span[@id='jupyterhub-logo']
+         Sleep  ${retries_delay}
```

Comment:
they will be executed if `${index} != ${retries}`, no?
only when `${index} == ${retries}` this will be skipped (no need to reload, we tried too many times)

Reply:
yes it will execute. I was thinking if it is fail also reload and wait. However i think above proposal of luca outside the loop is good.